### PR TITLE
Nav touchups

### PR DESCRIPTION
**Pull request by carlmjohnson**
This makes the nav more accessible and fixes issues with tabbing through the invisible items.

### CODE REVIEW

File: layouts/partials/tw/nav.html [hunk 1]
```diff
@@ -1,25 +1,33 @@
...
     >
       <svg class="block h-8 w-full sm:h-14 md:h-16 lg:h-20">
-        <title>Home</title>
-        <desc>Spotlight PA logo</desc>
+        <title>Spotlight PA Home</title>
         {{ partial "helper/svg.html" "banner" }}
       </svg>
     </a>
-  </h1>
+  </div>
```
Comment (jeffrummel) [inline layouts/partials/tw/nav.html:30]: What's the benefit of using a div over h1 here? Using an H1 to define the title in the page header has historically been a best-practice

Comment (carlmjohnson) [reply layouts/partials/tw/nav.html:30]: We use H1 for the page title already, so now we have two H1s, one that says Spotlight PA and one that says whatever the page is about.

File: layouts/partials/tw/nav.html [hunk 2]
```diff
@@ -1,25 +1,33 @@
...
 >
-  <h1
+  <div
+    class="hidden text-xs font-extrabold leading-tight tracking-wide sm:block sm:w-32 lg:w-52 lg:text-sm xl:text-base"
+  >
+    <p>Independent</p>
+    <p>Nonpartisan</p>
+    <p>Nonprofit</p>
+  </div>
+  <div
```
[the remark this answers is not in the document]
Comment (carlmjohnson) [reply layouts/partials/tw/nav.html:16]: I just did another test, and I think you're right. It's odd to hear the secondary tagline without context. I'll switch it back.

**Comment by carlmjohnson** [comment]
Margins should be fixed once it deploys.

**Comment by jeffrummel** [comment]
Screenshot 2023-06-08 at 9 16 00 AM


menu no longer aligns with max page width

**Comment by jeffrummel** [comment]
Screenshot 2023-06-08 at 09-34-31 Spotlight PA Pennsylvania news and investigations

There's also a positioning problem with expanded menu on larger screens

**Comment by carlmjohnson** [comment]
The company needs to buy me a larger laptop. Mine doesn't get that wide. 😆

**Comment by carlmjohnson** [comment]
Okay, I think that fixes it. Definitely feeling the CSS meme right now.

**Comment by carlmjohnson** [comment]
Seems okay with one more:

image

This what I get if I just add an element in the DOM inspector:

image

I can turn off the transition-translation if it's too distracting.

**Pull request merged**